content/knowledge/flare_architecure.md

```diff
@@ -0,0 +1,195 @@
+
+Signal providers exist in the off-chain to collect data, such as asset prices, and deliver it to the FTSO.
+As an example, a provider may collect the price of XRP/$ from three different exchanges, average the price and have this price estimate ready for the FTSO every voting round to submit.
+Signal providers on the Flare Network are responsible for delivering continuous and accurate data estimates to the FTSO to represent off-chain values on the Flare Network.
+Signal providers can provide their F-Asset and FLR votes to do this, but they also can receive delegated votes from other parties to perform this service on their behalf.
+Signal providers may take fees for this service.
+Additionally, the entire process is noncustodial (i.e. participants retain full control of their own data).
+The FTSO ensures the data is fair and accurate and makes it available to network applications.
+
+Additional info and some implementation details are available in [this example of price provider](https://github.com/flare-foundation/FTSO-price-provider).
+
+### Delegation
+
+Anyone can submit price data to the FTSO and vote for their price estimate.
+This may require some level to effort and technical skills to build such a provider.
+Most Spark and F-Asset holders can delegate their tokens to an FTSO provider who they trust will provide accurate price estimates.
+The delegation will be as simple as logging into a Flare wallet, choosing an FTSO provider, and finally selecting the amount tokens they wish to delegate.
+
+### Rewards
+
+Signal providers receive a reward for accurately voting on prices, as determined by the algorithm.
+
+The top and bottom 25% of prices are discarded, and the average of the remaining is used.
+The 50% of price providers who are closest to the price than receive rewards for their own tokens plus the percentage of fees, as defined by them on the smart contract.
+The default is 20%, and they can change it in 5% increments/decrements.
+
+The exact amount of compensation is  based on the number of delegated tokens minus a fee set by the FTSO provider.
+
+For example, Alice, Bob and Charlie choose to delegate their Spark tokens, 1000, 500 & 800 respectively for a total of 2,300 Spark delegated to FTSO Provider _X_.
+FTSO Provider _X_ submits their price estimate for the round and wins using their proprietary price estimate.
+Say, 100 Spark is awarded as compensation to FTSO Provider _X_.
+The provider will take 10 Spark as a fee (10%) and divide the rest of the 90 tokens to the delegators based on their stake ratio (delegators stake / total Spark delegated to FTSO Provider _X_).
+Thus, Alice gets 43.5% (39.15 Spark), Bob gets 21.7% (19.53 Spark) and Charlie gets 34.8% (31.32 Spark).
+Note: Only Spark delegators are compensated, F-Asset votes are not compensated by the FTSO.
+
+## F-Asset System
+
+The F-Asset System is an integral application on the Flare Network.
```

Comment:
Where the name F-Asset comes from? Flare Asset?

Reply:
I couldn't find the origin behind the name, but I think it is indeed short form of Flare Asset